### ISSUE TITLE
Patients heading includes additional data

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#5821]
## Area or Component

[Patients page](https://demo.simplereport.gov/app/patients/1?facility=8afbcdb3-1331-4583-9d53-d3bf13dd5bd8)

## Quick Summary

Patients page header includes "Showing x-y of z" text. 

## Accessibility Violation

[Heading not descriptive](https://docs.deque.com/issue-help/1.0.0/en/heading-not-descriptive)

## Action Requested

Move the additional text outside of the header.

## Screenshots
![image](https://github.com/CDCgov/prime-simplereport/assets/10108172/2bc19ca0-deef-4c4f-b350-eeb36abd3bc3)
